lua/nvim-treesitter/test_utils.lua

```diff
@@ -0,0 +1,38 @@
+local M = {}
+
+local assert = require('luassert')
+
+function M.set_buf_indent_opts(opts)
+    local optnames = {'tabstop', 'shiftwidth', 'softtabstop', 'expandtab'}
+    for _, opt in ipairs(optnames) do
+      if opts[opt] ~= nil then
+        vim.bo[opt] = opts[opt]
+      end
+    end
+end
+
+function M.indent_whole_file(file, opts)
+  assert.are.same(1, vim.fn.filereadable(file))
+
+  -- load reference file
+  vim.cmd(string.format('edit %s', file))
+  local reference = vim.api.nvim_buf_get_lines(0, 0, -1, true)
+
+  -- FIXME: why this is not set automatically?
```

Comment:
There's a FileType autocmd that *should* fix this but I still need to do this manually. Seems like it's not just me that still has this issue 😆

Reply:
Now when I disabled this it works as expected, so I guess it has already been fixed :)